Machine-Teaching-Examples/moab_control_type1a.ink

```diff
@@ -0,0 +1,133 @@
+###
+
+# MSFT Bonsai 
+# Copyright 2020 Microsoft
+# This code is licensed under MIT license (see LICENSE for details)
+
+# Moab Tutorial 1
+# This introductory sample demonstrates how to teach a policy for 
+# controlling a ball on the plate of a "Moab" hardware device. 
+
+# To understand this Inkling better, please follow our tutorial walkthrough: 
+# https://aka.ms/moab/tutorial1
+
+#Control type 1a definition: immediate reward and single state dependent constraints: in this problem |x| < Squareside & |y| < Squareside
```

Comment:
Was this Inkling snuck in?

Reply:
Ah! my mistake, I had previously worked on a control_type story and thi wa sthe inkling. Was not meant to be part of this branch. removed